server/svix-server/tests/e2e_attempt.rs

```diff
@@ -92,11 +92,6 @@ async fn test_list_attempted_messages() {
             if list.data.len() != 3 {
                 anyhow::bail!("list len {}, not 3", list.data.len());
             }
-
-            // Assert order
-            assert_eq!(list.data[0].msg, msg_3);
-            assert_eq!(list.data[1].msg, msg_2);
-            assert_eq!(list.data[2].msg, msg_1);
```

Comment:
This part of the test is being removed here since we can no longer guarantee the order of this response data, and the test fails periodically on ‘main’ due to race condition. If the order is important here and we want to keep this test, we can either

- Revert the change that caused this issue (writing messagedestination in worker instead of the handler), or
- Examine/modify MessageAttempt response and its dependency on messagedestination

Reply:
You don't need to verify order, but you can still check they all delivered.

Reply:
Now that I think about it, why isn't order consistent anymore?

Reply:
So this test started being an issue when we moved the messagedestination write from the create message handler, to the worker - messagedestinations is no longer necessarily written in the order messages were received. If you take a look at the `list_attempted_messages` handler, there is a dependency on messagedestination.

That's as far as I've gotten on the issue. The test started being an issue after the handler->worker write was committed.

Reply:
Ohh, sorry, I thought it was checking the *msgs*, not the attempted messages. All good. Yeah, we can no longer guarantee order, that's fine.